### PR TITLE
Add support for different passwords and region

### PR DESCRIPTION
**Pull request by ghost**
ManageIQ/CloudForms has mainly 3 types of passwords that can be set when initializing an appliance:

 - ***Application*** (default: admin/smartvm) - It can be updated using the API. The role needs to use the updated password after the task has been successfully executed.
 - ***PostgreSQL Database*** - It can be changed using `appliance_console_cli` command. On
   a ManageIQ appliance it is already set as the database is pre-configured.
 - ***Operating System*** (default: root/smartvm) - It can be updated with `echo <password> | passwd --stdin root`.

This PR also allows to set the region number with a default to `0`, which is the default on a ManageIQ appliance.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by ghost** [comment]
@machacekondra could you please review this PR ?

File: README.md [hunk 1]
```diff
@@ -87,10 +88,13 @@ The item in `miq_vm_nics` list of can contain following attributes:
 
 ManageIQ variables:
 
-| Name          | Default value     |  Description                                         |
-|---------------|-------------------|------------------------------------------------------|
-| miq_username  | admin             | The username used to login to ManageIQ. |
-| miq_password  | smartvm           | The password of user specific in username used to login to ManageIQ. |
+| Name              | Default value       |  Description                                         |
+|-------------------|---------------------|------------------------------------------------------|
+| miq_app_username  | admin               | The username used to login to ManageIQ. |
```
Comment (machacekondra) [inline README.md:93]: This is actually bacward compatibility break, we need to still support `miq_username`, but we can create alias for it as `miq_app_username`. And document just `miq_app_username` with note `miq_username` is alias. Same for password.

File: tasks/init_cfme.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   add_host:
     hostname: "{{ miq_ip_addr }}"
     ansible_host: "{{ miq_ip_addr }}"
-    ansible_password: "{{ miq_password }}"
+    ansible_password: smartvm
```
Comment (machacekondra) [inline tasks/init_cfme.yml:6]: "{{ miq_vm_root_password }}"

Comment (machacekondra) [reply tasks/init_cfme.yml:6]: Ah, no, this is right :)

**Review by machacekondra — COMMENTED** [review]
Thanks a lot for the patch, looks good, I have just one comment.

**Comment by ghost** [comment]
@machacekondra I have updated the patch to keep `miq_username` and `miq_password`, setting `miq_app_username` and `miq_app_password` with their values. Both can be overridden by extra_vars.

In addition, I have adding support to update the application password as requested in #21.

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks!

**Review by machacekondra — APPROVED** [review]
Thanks a lot!

**Comment by machacekondra** [comment]
Fixes: https://github.com/oVirt/ovirt-ansible-manageiq/issues/21

**Pull request merged**